server/mscalendar/settings_notifications.go

```diff
@@ -0,0 +1,137 @@
+package mscalendar
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/utils/settingspanel"
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+type notificationSetting struct {
+	title       string
+	description string
+	id          string
+	dependsOn   string
+	getCal      func(string) MSCalendar
+}
+
+func NewNotificationsSetting(getCal func(string) MSCalendar) settingspanel.Setting {
+	os := &notificationSetting{
+		title:       "Receive notifications of new events",
+		description: "Do you want to subscribe to new events and receive a message when they are created?",
+		id:          "notification_setting",
```

Comment:
`notification_setting` is pretty vague. We use the word "notification" for pretty much any DM sent to the user by a bot. What about `new_or_updated_event_setting`?